pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -23,7 +23,7 @@ const (
 	//
 	// Maintainers: Run ./Documentation/check-crd-compat-table.sh for each release
 	// Developers: Bump patch for each change in the CRD schema.
-	CustomResourceDefinitionSchemaVersion = "1.26.7"
+	CustomResourceDefinitionSchemaVersion = "1.26.8"
```

Comment:
aren't they going to bump the version if it gets changed in `main` anyway? 🤔

Reply:
I actually suspect it's an oversight in the original PR that they didn't bump the CRD schema version because I don't see how the changes would get picked up otherwise.

And I'm not sure the `-dd` suffix would change anything here because we need this semver check to be true for the Operator to upgrade the CRDs in the cluster:
https://github.com/DataDog/cilium/blob/314ca7baeff4f568ffc0bad95124e40665b1f88c/pkg/k8s/apis/cilium.io/client/v1beta1.go#L81
And I confirmed that this check ignores suffixes:
```go
func (vc *VersionCheckTestSuite) TestVersion(c *C) {
	ver1, _ := Version("1.26.7")
	ver2, _ := Version("1.26.7-dd0")
	c.Assert(ver1.LT(ver2), Equals, true)
}
```
```
c.Assert(ver1.LT(ver2), Equals, true)
Expected :bool = true
Actual   :bool = false

check_test.go:24
OOPS: 0 passed, 1 FAILED
--- FAIL: Test (0.00s)
```
So we'll have to update this check as well if we want to go this route (we'll need to make sure the check looks at `clusterVersion.Pre` field and that the `-dd` suffix is allowed in versions [here](https://github.com/DataDog/cilium/blob/314ca7baeff4f568ffc0bad95124e40665b1f88c/pkg/versioncheck/check.go#L56-L63))

Reply:
> aren't they going to bump the version if it gets changed in main anyway?

Well that's why I wondered if our bumping might conflict with theirs.

> LT() and 1.26.7 vs 1.26.7-dd0

Probably what you'd want then is to upgrade to 1.26.8-dd0 here then. But if we think this isn't necessary, that's fine too.

Reply:
oh indeed my bad, I did not realize the version bump was not part of their change!

Reply:
> I actually suspect it's an oversight in the original PR that they didn't bump the CRD schema

This is expected. IIRC they bump it only when they cut a new release to coalesce different changes into one bump.